src/anndata/_io/specs/methods.py

```diff
@@ -886,6 +862,25 @@ def chunk_slice(start: int, stop: int) -> tuple[slice | None, slice | None]:
         disk_mtx.append(elem[chunk_slice(chunk_start, chunk_stop)].compute())
 
 
+for array_type in [DaskArray, views.DaskArrayView]:
+    for group_type in [H5Group, ZarrGroup]:
+        for cupy_array_type, spec in [
+            (CupyArray, IOSpec("array", "0.2.0")),
+            (CupyCSCMatrix, IOSpec("csc_matrix", "0.1.0")),
+            (CupyCSRMatrix, IOSpec("csr_matrix", "0.1.0")),
+        ]:
+            _REGISTRY.register_write(group_type, (array_type, cupy_array_type), spec)(
+                write_cupy_dask
+            )
+        for scipy_sparse_type, spec in [
+            (sparse.csr_matrix, IOSpec("csr_matrix", "0.1.0")),
+            (sparse.csc_matrix, IOSpec("csc_matrix", "0.1.0")),
```

Comment:
we should support sparse arrays in dask also: https://github.com/scverse/anndata/issues/2085